src/runscripts.ron

```diff
@@ -81,7 +81,7 @@
 
     RunScriptSpec(
         key: "boltzmann:tsbuild",
-        value: "tsc --outDir target/release && cp boltzmann.js target/release",
+        value: "tsc && cp package.json target/release && cp package.json target/release/",
```

Comment:
does `boltzmann.js` get placed appropriately in `target/release`? I noted that I had to move it in there explicitly in some cases.